jx/JXML.js

```diff
@@ -54,3 +54,30 @@ JXML.deepMerge = function deepMerge(dst, src) {
 			dst[k] = v;
 	}
 }
+
+/**
+ * Same as deepMerge but returns data that actually changed.
```

Comment:
right now, returning data that actually changed is basically half-truth.

eg. JXML.mergeDiff({a: {b: {c:{}}} }, {a: 0}) will return  {a: 0}, when what has changed is also the removal of {b: {c: {}} }